src/main/java/de/dennisguse/opentracks/util/IntentDashboardUtils.java

```diff
@@ -87,6 +98,13 @@ public static void startDashboard(Context context, boolean isRecording, Track.Id
         clipData.addItem(new ClipData.Item(uris.get(MARKERS_URI_INDEX)));
         intent.setClipData(clipData);
 
+        if ((targetPackage != null) && (targetClass != null)) {
+            Log.i(TAG, "Starting dashboard activity with explicit intent (package=" + targetPackage + ", class=" + targetClass + ")");
+            intent.setClassName(targetPackage, targetClass);
+        } else {
+            Log.i(TAG, "Starting dashboard activity with generic intent (package=" + targetPackage + ", class=" + targetClass + ")");
```

Comment:
targetPackage and targetClass will not be used, so why log them?

Reply:
This is because of your previous comment above: "And a logcat output would be nice for debugging to check if actually something was received."

Reply:
Then I would move this into a separate log beforehand, but not important.